src/autorest-core/resources/default-configuration.md

```diff
@@ -111,14 +111,12 @@ pipeline:
     input: individual/transform
     scope: azure-validator-individual
   # validator written in TypeScript
-  swagger-document/azure-openapi-validator:
-    input:
-      - transform
-      - azure-validator # artificial predecessor in order to ensure order of messages for CI purposes
-  swagger-document/individual/azure-openapi-validator:
-    input: 
-      - transform
-      - azure-validator # artificial predecessor in order to ensure order of messages for CI purposes
+  swagger-document/azure-openapi-linter:
+    input: transform
+    scope: azure-validator-composed # artificial predecessor in order to ensure order of messages for CI purposes
```

Comment:
@olydis similar questions as above related the file in the next comment "pipeline.yaml"